src/Database/Type/BinaryType.php

```diff
@@ -51,7 +39,7 @@ public function toDatabase($value, Driver $driver)
      * @return resource|null
      * @throws \Cake\Core\Exception\Exception
      */
-    public function toPHP($value, Driver $driver)
+    public function fromDatabasetoPHP($value, Driver $driver)
```

Comment:
Not sure why this method needed to be renamed. Its also making the Datasource and Database packages have different methods if I'm understanding things correctly.